pubgate/utils/checks.py

```diff
@@ -23,7 +23,7 @@ def user_check(handler=None):
     async def wrapper(request, *args, **kwargs):
         user = await User.find_one(dict(name=kwargs["user"].lower()))
         if not user:
-            raise exceptions.NotFound("User not found")
```

Comment:
actually, sainc.exceptions were a json response in previous versions. going to fix them all)